sqlgen/db.go

```diff
@@ -32,10 +30,8 @@ func NewDB(conn *sql.DB, schema *Schema) *DB {
 		Many: func(ctx context.Context, items []interface{}) ([]interface{}, error) {
 			table := items[0].(*BaseSelectQuery).Table
 
-			if span := opentracing.SpanFromContext(ctx); span != nil {
-				span, ctx = opentracingkit.MaybeStartSpanFromContext(ctx, fmt.Sprintf("thunder.sqlgen.BatchQuery(%s)", table.Name))
-				defer span.Finish()
-			}
+			span, ctx := opentracingkit.MaybeStartSpanFromContext(ctx, fmt.Sprintf("thunder.sqlgen.BatchQuery(%s)", table.Name))
```

Comment:
Quick fly by comment:
- We should also tag the number of items being queried and the number of rows returned.